core/extensions.rs

```diff
@@ -195,6 +195,23 @@ macro_rules! ops {
   }
 }
 
+#[macro_export]
+macro_rules! or {
+  ($e:expr, $fallback:expr) => {
+    $e
+  };
+  (, $fallback:expr) => {
+    $fallback
+  };
+}
+
+#[macro_export]
+macro_rules! concat_slices {
```

Comment:
Is this used? I wonder if it might be more appropriate to call it "concat_array_literals" instead, however.

Reply:
No it's not used, I forgot to remove it